### PR TITLE
Delete gce ingress test configs

### PR DESCRIPTION
**Pull request by bprashanth**
The difference between gce and gci is insignificant for ingress and we're running too many shareded ingress suites to be of any practical value. Delete the old suites and just keep gci-gce/gke-{1.4,1.5,HEAD} debian-gce/gke-{HEAD}

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1164)

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml [hunk 1]
```diff
@@ -197,10 +197,6 @@
         job-name: ci-kubernetes-e2e-gce-serial-release-1.2
         frequency: 'H H/6 * * *' # 4 times a day for older jobs.
         trigger-job: 'kubernetes-build-1.2'
-    - kubernetes-e2e-gce-ingress-release-1.2:
```
Comment (fejta) [inline jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml]: Will you also delete the corresponding test-infra/jobs/ci-kubernetes-e2e-gce-ingress-release-1.2.sh (etc) jobs?

**Comment by fejta** [comment]
/lgtm

Once this passes unit tests and the .sh files are deleted feel free to commit.

**Comment by bprashanth** [comment]
Tests are green, lgtm?

**Pull request merged**